lib/Terminal/ConnectionToken.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Stripe\Terminal;
+
+/**
+ * Class ConnectionToken
+ *
+ * @property string $connection_token
```

Comment:
Minor, but can we change this to the new `$secret` property?

Reply:
Yep!